tests/e2e/test-lint-r.R

```diff
@@ -0,0 +1,7 @@
+rhino::lint_r()
+# Create bad scripts and test if formatting returns the expected result
+test_file_path <- fs::path("app", "logic", "bad-style.R")
+cat("bad_object_style=12", file = test_file_path)
+testthat::expect_error(rhino::lint_r())
+# Clean up
+file.remove(test_file_path)

```

Comment:
All test scripts should take this simple approach of creating a file, running the function, using `testthat::expect_error` to confirm an error.